includes/class-wau-pro-features.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * WooCommerce Addon Uploads Pro Features Class
+ *
+ * Contains all the pro features and other plugin information
+ *
+ * @author      Dhruvin Shah
+ * @package     WooCommerce Addon Uploads
+ */
+
+if ( ! class_exists( 'Wau_Pro_Features' ) ) {
+
+	/**
+	 * This class contains pro features of WooCommerce addon uploads
+	 */
+	class Wau_Pro_Features {
+
+		/**
+		 * This function takes a callback to enqueue scripts
+		 */
+		public function __construct() {
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts_and_styles' ) );
+		}
+
+		/**
+		 * This functions enqueues styles
+		 */
+		public function enqueue_scripts_and_styles() {
+
+			wp_register_style( 'pro-feature-css', plugins_url( '../assets/css/wau_features.css', __FILE__ ), array(), '1.6.0', 'all' );
+			wp_enqueue_style( 'pro-feature-css' );
+
+		}
+
+		/**
+		 * This function gets json data and show that as in html template
+		 */
+		public static function pro_features_callback() {
+			$str  = wp_remote_get( 'https://sandbox.imaginate-solutions.com/lite_assets/addon/addon_lite.json' );
+			$json = json_decode( $str['body'], true );
```

Comment:
The above line must be replaced with the following:

```
if ( ! is_wp_error( $str ) ) {
	$json = json_decode( wp_remote_retrieve_body( $str ) );
}
```

we must have proper validations in place